docs/wrappers.md

```diff
@@ -1,14 +1,78 @@
-experiments. 
-These can be found here: https://github.com/openai/baselines/blob/master/baselines/common/atari_wrappers.py
+`FilterObservation(env, filter_keys)` [text]
+* Needs review (including for good assertion messages and test coverage)
+
+`FlattenObservation(env)` [text]
+* Needs good asseration messages
+
+`FrameStack(env, num_stack, lz4_compress=False)` [text]
+* Needs review (including for good assertion messages and test coverage)
+
+`GrayScaleObservation(env)` [text] 
+* Needs R/G/B channel argument added like supersuit wrapper
+* Needs review (including for good assertion messages and test coverage)
+* Needs CV2 dependency replaced with SuperSuit's way of doing full grey scaling: https://github.com/PettingZoo-Team/SuperSuit/blob/master/supersuit/utils/basic_transforms/color_reduction.py
+
+`PixelObservationWrapper(pixels_only=True, render_kwargs=None, pixel_keys=("pixels",))` [text]
+* Needs review (including for good assertion messages and test coverage)
+
+`RescaleObservation(env, min_obs, max_obs`) [text]
+* Bring over from Supersuit or from https://github.com/openai/gym/pull/1635
+
+`ResizeObservation(env, shape)` [text]
+* Needs review (including for good assertion messages and test coverage)
+* Switch from CV2 to Lycon2 once released
+
+`TimeAwareObservation(env)` [text]
+* Needs review (including for good assertion messages and test coverage)
+
+## Action Wrappers
+
+`ClipAction(env)` [text]
+* Needs review (including for good assertion messages and test coverage)
+
+`ChangeDtype(env, dtype)` [text]
+* Bring over from SuperSuit
+
+`RescaleAction(env, min_action, max_action)` [text]
+* Needs review (including for good assertion messages and test coverage)
+
+`StickyActions(env, repeat_action_probability)` [text]
+* Create as separate wrapper from Atari/bring over from SuperSuit
+
+## Reward Wrappers
+
+`ClipReward(env, lower_bound-1, upper_bound=1)` [text]
+* Bring over from SuperSuit
+
+## Lambda Wrappers
+
+`ActionLambda(env, change_action_fn, change_space_fn)` [text]
+* Bring over from SuperSuit
+
+`ObservationLambda(env, observation_fn, observation_space_fn)` [text]
+* Bring over from SuperSuit, replaces TransformObservation
+
+`RewardLambda(env, reward_fn)` [text]
+* Bring over from SuperSuit, replaces TransformReward
+
+## Other
+`AtariPreprocessing(env, noop_max=30, frame_skip=4, screen_size=84, terminal_on_life_loss=False, grayscale_obs=True, grayscale_newaxis=False, scale_obs=False)` [text]
+* Needs review (including for good assertion messages and test coverage)
+
+`RecordEpisodeStatistic(env)` [text]
+* Needs review (including for good assertion messages and test coverage)
+
+`RecordVideo(env, ...)` [text]
+* https://github.com/openai/gym/pull/2300
+
+`TimeLimit(env, max_episode_steps)` [text]
+* Needs review (including for good assertion messages and test coverage)
+
+Some sort of vector environment conversion wrapper needs to be added here, this will be figured out after the API is changed.
```

Comment:
What would be the advantage of having a wrapper for converting an environment to a vector environment compared to using `SyncVectorEnv`/`AsyncVectorEnv` directly?

I don't think that would be appropriate, for example let's say you have a wrapper `Vectorize`, and then you do
```python
env = gym.make('CartPole-v1')
envs = Vectorize(env)
```
Then how do you define [`envs.unwrapped`](https://github.com/openai/gym/blob/master/gym/core.py#L271-L273) for parallel environments? Do you always run a version of the wrapped environment in the main process (which is unnecessary most of the time)? Or do you run that environment in a subprocess (for example, the first process, but that probably rules out `envs.unwrapped.unwrapped`)?